blog/2024-04-26-license-questions.mdx

```diff
@@ -0,0 +1,68 @@
+---
+slug: license-questions
+title: License Questions
+authors: [parker]
+tags: [panel, license]
+hide_table_of_contents: true
+---
+
+As some of you know, people have been asking us questions about when you need a license or not.
+
+I asked for people to get me questions they wanted clarifications on and here are some answers. 
+
+## modification questions
+
+1.  Q: What are the plans for the theming part of Pelican Panel and what will the front end look like?
+
+    A: Using [Filament themes](https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme) placed in `/resources/css/filament` will work and don't need a license, 
+    and so long as no files that are included with the panel are modified in the process, you do not need a commercial license.
+
+:::info
+    A theme selector will eventually be implemented.
+:::
+:::warning
+    Themes are unable to change the layout of the panel, but plugins can.
```

Comment:
```suggestion
    Themes are unable to heavily change the layout of the panel because they are limited to CSS. However plugins can completely and more easily rearrange components and add new ones and pages.
```